From 33c0229109d8f6b76a0ba0985e640f7892a0c4bf Mon Sep 17 00:00:00 2001
From: Chuck Grindel <chuck.grindel@gmail.com>
Date: Wed, 7 Jun 2023 11:24:48 -0600
Subject: [PATCH] Do not exclude files that are needed for compilation

---
 Package.swift | 2 --
 1 file changed, 2 deletions(-)

diff --git a/Package.swift b/Package.swift
index 2ebd25d..ba31d5c 100644
--- a/Package.swift
+++ b/Package.swift
@@ -53,8 +53,6 @@ let package = Package(
             "config.h.in",
             "CMakeLists.txt",
             "cmark-gfm_version.h.in",
-            "case_fold_switch.inc",
-            "entities.inc",
           ],
           cSettings: cSettings
         ),
-- 
2.41.0

